superset-frontend/src/SqlLab/components/SqlEditor.jsx

```diff
@@ -154,6 +154,18 @@ const defaultProps = {
   scheduleQueryWarning: null,
 };
 
+// Detects the user's OS through the browser
+export function detectOS() {
```

Comment:
use arrow function

Reply:
Sorry for jumping in, but why arrow function? I don't remember we are enforcing this style. IMO `function` reads cleaner, especially when used in exports.

Reply:
Using a named function also has the benefit of the name showing up in stack traces